.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```

Comment:
A couple changes to make here:
1. This file should be removed from version control.
2. `.swiftpm` should be added to the [gitignore](https://github.com/kyleweiner/KWStepper/blob/46bd5dbd4adb062e37992f62ad5f355528b4cbf1/.gitignore#L21) file.